### ISSUE TITLE
Resolve accessibility issue with Breadcrumb 'Home' link

### ISSUE THREAD

**Issue by RhysMAdey** [DiamondLightSource/sci-react-ui#62]
Accessibility flags:

![Image](https://github.com/user-attachments/assets/9af8f3f5-4f23-41ef-8f27-4ac20594657a)

I believe the fix should be as simple as adding an 'aria-label' to the 'Home' link
